source/assets/stylesheets/patterns/_guides-list.scss

```diff
@@ -0,0 +1,46 @@
+.guides-list {
+  @extend %section;
+  padding-top: 0;
+
+  &-heading {
+    display: inline-block;
+
+    &::after {
+      content: "→";
+      opacity: 0;
+      padding-left: $small-spacing / 2;
+    }
+  }
+
+  p {
+    color: $white;
+    margin-bottom: 0;
+  }
+}
+
+.guides-link {
+  border-radius: $base-border-radius;
+  color: $action-color;
+  display: block;
+  margin-bottom: $base-spacing;
+  padding: $small-spacing;
+
+  &:hover {
+    @include transition(background-color 0.25s ease-in-out);
```

Comment:
FYI: `transition` has been available without vendor prefix in pretty much all browsers for a while now

Reply:
@killercup a sign that I haven't written CSS in ages 😅. Feel free to trim.